example/quantization/imagenet_gen_qsym_mkldnn.py

```diff
@@ -234,6 +242,12 @@ def save_params(fname, arg_params, aux_params, logger=None):
                                'mobilenet0_pool0_fwd']
         if exclude_first_conv:
             excluded_sym_names += ['mobilenet0_conv0_fwd']
+    elif args.model == 'mobilenetv2_1.0':
+        rgb_mean = '123.68,116.779,103.939'
```

Comment:
is there any exception that the rgb_mean dna std is not the same? otherwise repeatively coding it looks redundant

Reply:
Agree with @zhreshold, along with enabling more models, we don't need to show how to reproduce each one since most of the command is very similar. We need to define a template for the user to reproduce any existed models for classification networks.

Reply:
@zhreshold @pengzhao-intel agree, I'll refactor this script along with enabling more models next time.